readme.md

```diff
@@ -26,7 +33,7 @@ Describe at a high level how we use jQuery to submit a form via AJAX.
 
 Your Answer:
 ```text
-
+We use jQuery to asynchornously response to the requests.
```

Comment:
Again, close here. We use jQuery to make a request and handle the asynchronous response